spec/rails_helper.rb

```diff
@@ -25,6 +25,7 @@
   config.use_transactional_fixtures = true
 
   config.before(:suite) do
+    DatabaseCleaner.allow_remote_database_url = ENV['TRAVIS'].present?
```

Comment:
Same deal as the travis change, this lets DB cleaner run against local socket auth during test.